tests/src/Context/BeanContext.php

```diff
@@ -30,6 +30,13 @@ class BeanContext implements Context {
   protected $random;
 
   /**
+   * The list of block types created before or during a scenario starts.
+   *
+   * @var array
+   */
+  protected $blockType = array();
```

Comment:
Why? bundle is valid for node types and other existing entity bundles. By using blockType, we point clearly Bean bundles.
And in the Bean entity info declaration, the bundle key = 'type'...